anki/template/template.py

```diff
@@ -200,20 +205,27 @@ def render_unescaped(self, tag_name=None, context=None):
 
     def clozeText(self, txt, ord, type):
         reg = clozeReg
-        if not re.search(reg%ord, txt):
+        if not re.search(reg % ord, txt):
             return ""
+
         def repl(m):
             # replace chosen cloze with type
+            noSound = removeClozeSound(m.group(3))
```

Comment:
I'm sure it wouldn't.

This is actually about the sounds in a cloze.  If you create a multi-cloze card,  and you put a sound in each cloze, either in the answer, or in the hint, when you use that card it will start playing every sound from every cloze until it finishes.  Likewise when you get the answer.

What this does is remove the sounds for all clozes except the ones belonging to the current card render.  So if you have a multi-close card that generates 5 cloze cards.  You won't hear the audio from the other 4 clozes.  Only the clozes for the card you are on.

Without this chanage multi cloze cards and audio just don't work.